[2015-08-27 12:48:50] request.INFO: Matched route "portada". {"route_parameters":{"_controller":"App\\CorredoresRiojaBundle\\Controller\\DefaultController::indexAction","_locale":"es","_route":"portada"},"request_uri":"http://localhost:8000/app.php/es/portada"} []
[2015-08-27 12:48:50] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-08-27 12:48:50] request.CRITICAL: Uncaught PHP Exception PDOException: "SQLSTATE[42000] [1049] Unknown database 'corredoresporlarioja'" at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php line 40 {"exception":"[object] (PDOException(code: 1049): SQLSTATE[42000] [1049] Unknown database 'corredoresporlarioja' at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:40)"} []
[2015-09-09 11:47:05] request.INFO: Matched route "organizacion_portada". {"route_parameters":{"_controller":"App\\CorredoresRiojaBundle\\Controller\\OrganizacionController::portadaAction","_locale":"en","_route":"organizacion_portada"},"request_uri":"http://localhost:8000/app.php/en/organizations"} []
[2015-09-09 11:47:05] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2015-09-09 11:47:05] security.DEBUG: User was reloaded from a user provider. {"username":"1234","provider":"App\\CorredoresRiojaBundle\\Security\\CorredorUserProvider"} []
[2015-09-09 11:47:05] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 1 passed to App\CorredoresRiojaDomain\Repository\Infrastructure\DoctrineCarreraRepository::buscaCarrerasDisputadasOrganizadasPor() must be an instance of App\CorredoresRiojaDomain\Model\Organizacion, null given, called in /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaBundle/Controller/OrganizacionController.php on line 20 and defined" at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaDomain/Repository/Infrastructure/DoctrineCarreraRepository.php line 47 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to App\\CorredoresRiojaDomain\\Repository\\Infrastructure\\DoctrineCarreraRepository::buscaCarrerasDisputadasOrganizadasPor() must be an instance of App\\CorredoresRiojaDomain\\Model\\Organizacion, null given, called in /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaBundle/Controller/OrganizacionController.php on line 20 and defined at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaDomain/Repository/Infrastructure/DoctrineCarreraRepository.php:47)"} []
[2015-09-09 11:47:05] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2015-09-09 11:47:10] request.INFO: Matched route "organizacion_portada". {"route_parameters":{"_controller":"App\\CorredoresRiojaBundle\\Controller\\OrganizacionController::portadaAction","_locale":"en","_route":"organizacion_portada"},"request_uri":"http://localhost:8000/app.php/en/organizations"} []
[2015-09-09 11:47:10] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2015-09-09 11:47:10] security.DEBUG: User was reloaded from a user provider. {"username":"1234","provider":"App\\CorredoresRiojaBundle\\Security\\CorredorUserProvider"} []
[2015-09-09 11:47:10] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 1 passed to App\CorredoresRiojaDomain\Repository\Infrastructure\DoctrineCarreraRepository::buscaCarrerasDisputadasOrganizadasPor() must be an instance of App\CorredoresRiojaDomain\Model\Organizacion, null given, called in /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaBundle/Controller/OrganizacionController.php on line 20 and defined" at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaDomain/Repository/Infrastructure/DoctrineCarreraRepository.php line 47 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to App\\CorredoresRiojaDomain\\Repository\\Infrastructure\\DoctrineCarreraRepository::buscaCarrerasDisputadasOrganizadasPor() must be an instance of App\\CorredoresRiojaDomain\\Model\\Organizacion, null given, called in /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaBundle/Controller/OrganizacionController.php on line 20 and defined at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaDomain/Repository/Infrastructure/DoctrineCarreraRepository.php:47)"} []
[2015-09-09 11:47:10] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2015-09-09 11:47:24] request.INFO: Matched route "organizacion_portada". {"route_parameters":{"_controller":"App\\CorredoresRiojaBundle\\Controller\\OrganizacionController::portadaAction","_locale":"en","_route":"organizacion_portada"},"request_uri":"http://localhost:8000/app.php/en/organizations"} []
[2015-09-09 11:47:24] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2015-09-09 11:47:24] security.DEBUG: User was reloaded from a user provider. {"username":"1234","provider":"App\\CorredoresRiojaBundle\\Security\\CorredorUserProvider"} []
[2015-09-09 11:47:24] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 1 passed to App\CorredoresRiojaDomain\Repository\Infrastructure\DoctrineCarreraRepository::buscaCarrerasDisputadasOrganizadasPor() must be an instance of App\CorredoresRiojaDomain\Model\Organizacion, null given, called in /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaBundle/Controller/OrganizacionController.php on line 20 and defined" at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaDomain/Repository/Infrastructure/DoctrineCarreraRepository.php line 47 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to App\\CorredoresRiojaDomain\\Repository\\Infrastructure\\DoctrineCarreraRepository::buscaCarrerasDisputadasOrganizadasPor() must be an instance of App\\CorredoresRiojaDomain\\Model\\Organizacion, null given, called in /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaBundle/Controller/OrganizacionController.php on line 20 and defined at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/src/App/CorredoresRiojaDomain/Repository/Infrastructure/DoctrineCarreraRepository.php:47)"} []
[2015-09-09 11:47:24] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2015-09-14 11:30:41] request.INFO: Matched route "portada". {"route_parameters":{"_controller":"App\\CorredoresRiojaBundle\\Controller\\DefaultController::indexAction","_locale":"es","_route":"portada"},"request_uri":"http://localhost:8000/app.php/es/portada"} []
[2015-09-14 11:30:41] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-09-14 11:30:41] request.CRITICAL: Uncaught PHP Exception PDOException: "SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php line 40 {"exception":"[object] (PDOException(code: 1045): SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:40)"} []
[2015-09-14 11:55:19] request.INFO: Matched route "admin". {"route_parameters":{"_controller":"JavierEguiluz\\Bundle\\EasyAdminBundle\\Controller\\AdminController::indexAction","_route":"admin"},"request_uri":"http://localhost:8000/app.php/admin/?action=list&entity=Corredor"} []
[2015-09-14 11:55:19] security.DEBUG: Read existing security token from the session. {"key":"_security_backend"} []
[2015-09-14 11:55:19] security.DEBUG: User was reloaded from a user provider. {"username":"admin","provider":"Symfony\\Component\\Security\\Core\\User\\InMemoryUserProvider"} []
[2015-09-14 11:55:19] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)") in "@EasyAdmin/default/list.html.twig" at line 129." at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/app/cache/prod/classes.php line 5168 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)\") in \"@EasyAdmin/default/list.html.twig\" at line 129. at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/app/cache/prod/classes.php:5168, PDOException(code: 1045): SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:40)"} []
[2015-09-14 11:55:19] security.DEBUG: Stored the security token in the session. {"key":"_security_backend"} []
[2015-09-14 11:55:37] request.INFO: Matched route "admin". {"route_parameters":{"_controller":"JavierEguiluz\\Bundle\\EasyAdminBundle\\Controller\\AdminController::indexAction","_route":"admin"},"request_uri":"http://localhost:8000/app.php/admin/?action=list&entity=Corredor"} []
[2015-09-14 11:55:38] security.DEBUG: Read existing security token from the session. {"key":"_security_backend"} []
[2015-09-14 11:55:38] security.DEBUG: User was reloaded from a user provider. {"username":"admin","provider":"Symfony\\Component\\Security\\Core\\User\\InMemoryUserProvider"} []
[2015-09-14 11:55:38] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)") in "@EasyAdmin/default/list.html.twig" at line 129." at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/app/cache/prod/classes.php line 5168 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)\") in \"@EasyAdmin/default/list.html.twig\" at line 129. at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/app/cache/prod/classes.php:5168, PDOException(code: 1045): SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:40)"} []
[2015-09-14 11:55:38] security.DEBUG: Stored the security token in the session. {"key":"_security_backend"} []
[2015-09-14 11:57:21] request.INFO: Matched route "anadir_carrera". {"route_parameters":{"_controller":"App\\CorredoresRiojaBundle\\Controller\\OrganizacionController::nuevaCarreraAction","_locale":"es","_route":"anadir_carrera"},"request_uri":"http://localhost:8000/app.php/es/organizaciones/nueva_carrera"} []
[2015-09-14 11:57:21] security.DEBUG: Read existing security token from the session. {"key":"_security_organizacion"} []
[2015-09-14 11:57:21] security.DEBUG: User was reloaded from a user provider. {"username":"matute@gmail.com","provider":"App\\CorredoresRiojaBundle\\Security\\OrganizacionUserProvider"} []
[2015-09-14 11:57:21] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occurred while executing 'INSERT INTO carrera (id, nombre, descripcion, fechacelebracion, distancia, fechalimiteinscripcion, numeromaximoparticipantes, imagen, slug, organizador_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [2710, "fafa", "Prueaba", "2010-01-01 00:00:00", 0, "2010-01-01 00:00:00", 50, null, "fafa", 4640]:  SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'imagen' cannot be null" at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 91 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO carrera (id, nombre, descripcion, fechacelebracion, distancia, fechalimiteinscripcion, numeromaximoparticipantes, imagen, slug, organizador_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [2710, \"fafa\", \"Prueaba\", \"2010-01-01 00:00:00\", 0, \"2010-01-01 00:00:00\", 50, null, \"fafa\", 4640]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1048 Column 'imagen' cannot be null at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:91, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'imagen' cannot be null at /home/jonathan/symfonyprojects/corredoresporlarioja/corredoresporlarioja/vendor/doctrine/dbal/lib/Doctrine/DBAL/Statement.php:165)"} []
[2015-09-14 11:57:21] security.DEBUG: Stored the security token in the session. {"key":"_security_organizacion"} []
